make-rules/ips.mk

```diff
@@ -151,6 +151,23 @@ ifneq ($(wildcard $(HISTORY)),)
 HISTORICAL_MANIFESTS = $(shell $(NAWK) -v FUNCTION=name -f $(GENERATE_HISTORY) < $(HISTORY))
 endif
 
+define ips-print-depend-require-rule
+$(shell cat $(1) $(WS_TOP)/transforms/print-depend-require |\
+	$(PKGMOGRIFY) $(PKG_OPTIONS) /dev/fd/0 |\
+	sed -e '/^$$/d' -e '/^#.*$$/d' | sort -u)
+endef
+
+define ips-print--depend-require-versioned-rule
+$(foreach v,$($(1)V_VALUES),\
```

Comment:
Is there a typo here: ...print--depend... (double hyphen)?
In the foreach loop it is written with a single hyphen.

Reply:
You are right, fixed.